### DISCUSSION TITLE
Empty state on a disabled section

### DISCUSSION THREAD

**Discussion by lgalloherran** [twilio-labs/paste#2445] [Office hours] [answered]
Hi Dsys folks!
I'm working on a use case for the messaging configuration section on the Phone Number configuration page. [ Link to the proposed design here. ](https://www.figma.com/proto/Lb81LexUpQATGMGH5ryRzu/PN-RC-toll-free-code-red?page-id=65%3A75032&node-id=65%3A75033&viewport=637%2C417%2C0.41&scaling=min-zoom)

**Use case:** If a customer buys or has a Toll free number, they will need to submit a Toll Free verification request before being able to use the messaging capability of this number. 

**Proposed actions:** Hide the messaging configuration options and replace them with an empty state that tells users what to do in order to enable the messaging configuration.

**Question:** Is this the right use case to use an empty state? If not, what would be a better component to use in this case?

**Comment by loreina** [comment]
Hi @lgalloherran, I circled this back to the team and we'll get back to you soon!

**Comment by loreina** [comment, marked as answer]
Hi @lgalloherran, thanks for being patient while we looked into this. 

It feels like there's some redundant messaging on the page. There's a warning alert and a warning icon with text for adding a messaging address, though it's unclear if those are connected. There's also a warning alert and an empty state for toll-free verification that also don't feel connected. I'm also unsure if the empty state is meant to indicate a warning, as it doesn't look like it is.

Have you explored other options for communicating the warnings? I wonder if the alerts could be moved from the top of the page to be inline with the relevant sections on the page? If you're worried about the warnings being missed, one idea to try is adding a warning icon besides "Configure" in the tabs.

Hope this helps! Let us know if you have any other questions. :)

**Reply by serifluous** [reply]
@lgalloherran Can you share what additional questions or concerns you have so we can help prep for office hours? Thanks!

Have you tried the options we shared above? That could help us make a decision in office hours on a good solution to move forward with so we can compare a few options.

**Comment by lgalloherran** [comment]
Hey, 
I'll explore the warning icon near the configure tab and see if it's visible enough. I was exploring what is may seem as a redundant warning and empty state because this page has a lot of important content below the fold, so it's not obvious to find important information right away when the page is loaded. 
Also, the empty state is there to indicate that the messaging configuration is blocked until the user performs the verification.

**Comment by lgalloherran** [comment]
I think  with this I can work for now, so there's no need to go to the OH tomorrow. 
Thanks for your time!

**Reply by serifluous** [reply]
Ok, thanks!
